python/semantic_kernel/planners/stepwise_planner/stepwise_planner.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         prompt_config.template = prompt_template
 
         self._system_step_function = self.import_function_from_prompt(kernel, "StepwiseStep", prompt_config)
-        self._native_functions = self._kernel.import_plugin_from_object(self, RESTRICTED_PLUGIN_NAME)
+        self._native_functions = self._kernel.add_plugin(self, RESTRICTED_PLUGIN_NAME)
```

Comment:
`self._native_functions` will now be None if we don't return the plugin from the kernel.